src/util/gradleUtil.ts

```diff
@@ -142,7 +142,7 @@ export function findChildGradleProjects(buildFile: any, settingsFile: any): Grad
  * @param gradlePath build.gradle file
  * @param workspaceFolder workspace of current project
  */
-export async function getGradleTestReport(gradlePath: any, workspaceFolder: vscode.WorkspaceFolder): Promise<string> {
+export async function getGradleTestReport(gradlePath: any, workspaceFolder: string): Promise<string> {
```

Comment:
Should rename this variable from `workspaceFolder` to something more appropriate now that we aren't using `vscode.WorkspaceFolder`. Same comment applies to `findCustomTestReport` and `helperUtil` methods below

Reply:
done